content/en/docs/deployment/device-provisioning-service.md

```diff
@@ -0,0 +1,242 @@
+---
+title: 'Device Provisioning Service'
+description: 'Running Device Provisioning Service on Kubernetes'
+docsOthersDisplay: true
+date: '2023-05-18'
+categories: [deployment, kubernetes]
+keywords: [deployment, kubernetes, k8s, helm, chart]
+weight: 2
+---
+
+![Deployment diagram](../static/dps-deployment-diagram.drawio.svg "medium-zoom-image")
+
+To enable the Device Provisioning Service, you need to extend the hub configuration.
+
+{{< note >}}
+Before deploying the Device Provisioning Service on Kubernetes, make sure to follow the steps in [Hub](../hub) first. Then apply the changes from this page to the configuration. Once done, you can deploy the hub with the Device Provisioning Service.
+{{< /note >}}
+
+For Device Provisioning Service, all configuration values are documented [here](https://github.com/plgd-dev/device-provisioning-service/blob/main/charts/device-provisioning-service/README.md#values).
+
+## Device provider for Device Provisioning Service
+
+When a device is successfully attested against an enrollment group by the Device Provisioning Service, the device's owner needs to be set. The owner is configured in the enrollment group. Therefore, the Device Provisioning Service needs to obtain the authorization code from the OAuth service, which is associated with the owner. When the device contacts the hub, the hub will exchange the authorization code for a JWT access token that contains the owner claim with the owner.
+
+### Client credential flow
+
+In this flow, the OAuth client is associated with the owner. The client must set the owner claim with the owner in the JWT access token.
+
+When using the `clientCredentials` flow, you cannot use `sub` because the owner will be the OAuth client. Therefore, you need to use another claim to identify the owner. We need to override default the hub configuration's section with  new `ownerClaim`, dps and web oauth clients:
+
+```yaml
+global:
+  ownerClaim: "https://plgd.dev/owner" # for mockoauthserver
+  oauth:
+    device:
+    - name: "plgd.dps"
+      clientID: my-client-id
```

Comment:
Is `my-client-id` intended to be a variable or a string? Lets keep the format consistent in examples,

if it's a string then it should be in quotes:
```yaml
clientID: "my-client-id"
```


if it's a variable then lets keep it consistent with line 73 and make it:
```yaml
clientID: "<my-client-id>"
```

or 

```yaml
clientID: "<MY_CLIENT_ID>"
```

Same for `clientSecret.`

Reply:
fixed